content/docs/details/updating-the-ui/index.md

```diff
@@ -104,6 +104,32 @@ Your new page will appear as part of the visible items, both in the Sidebar Sett
 
 ![Edit Sidebar Page - after](./edit-sidebar-page-after.jpeg)
 
+### Sidebar Links
+
+You can create links to other pages in Roadie for quick access or to external websites using the iFrame plugin. 
```

Comment:
This is not quite correct IMO. This is embedding the external page into the application, it's not linking anywhere

Reply:
Maybe that's an engineering distinction though? Our customers ask - how can I add a link to the website. I'm trying to answer that and make it searchable.

Reply:
I don't think so. That is not a link to a website, it's a link to Roadie internal path